theano/gof/opt.py

```diff
@@ -1770,6 +1771,8 @@ def process_node(self, fgraph, node, lopt=None):
                 raise
         if replacements is False or replacements is None:
             return False
+        if replacements is True:
+            return True
```

Comment:
What does that mean for `replacements` to be `True`? This is not documented in http://deeplearning.net/software/theano/extending/optimization.html#global-and-local-optimizations, nor in the docstring of `LocalOptimizer.transform` https://github.com/nouiz/Theano/blob/crash_gpu/theano/gof/opt.py#L1000